dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
```

Comment:
OK, in that case the other methods need to be renamed to avoid further confusion:

- `software_versions` => `package_versions`
- `package_versions` => `package_revisions`
- `latest_version` => `latest_revision`